protected/humhub/modules/friendship/widgets/FriendshipButton.php

```diff
@@ -37,7 +38,7 @@ private function getDefaultOptions()
     {
         return [
             'friends' => [
-                'title' => '<span class="glyphicon glyphicon-ok"></span>&nbsp;&nbsp;' . Yii::t('FriendshipModule.base', 'Friends'),
+                'title' => Icon::get('check') . Yii::t('FriendshipModule.base', 'Friends'),
```

Comment:
@yurabakhtin not tested on my side, but don't we need a space between the icon and the text?
`Icon::get('check') . ' ' . Yii::t('FriendshipModule.base', 'Friends'),`

Reply:
@marc-farre No need the space, because there is a space by style margin:
- https://github.com/humhub/humhub/blob/develop/static/scss/_button.scss#L15-L19
- https://github.com/humhub/humhub/blob/develop/protected/humhub/widgets/bootstrap/Button.php#L243-L245
<img width="842" height="686" alt="icon-margin" src="https://github.com/user-attachments/assets/e8759300-60cd-45d3-a911-dee1e32fee27" />